### PR TITLE
Exclude menus from lists (13. Content Structure - Visually Apparent Lists)

### PR DESCRIPTION
**Pull request by kengdoj**
[Preview Changes](https://federalist-78ccea55-7a06-447c-aabf-bd620b207b74.app.cloud.gov/preview/atbcb/icttestingbaseline/kengdoj-listmenu/13Structure/#limitations-assumptions-or-exceptions)

Closes https://github.com/atbcb/ICTTestingBaseline/issues/373

### CODE REVIEW

File: _baselines/13Structure.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ order-number: 14
 -   The heading level 1 on a page is not required to match the page title.
 -   The order of heading levels may not always be in sequence but may be valid as it relates to the visual structure/importance communicated by visible headings on the page. For example, an `<h2>` heading may be used for a navigation structure that precedes an `<h1>` title on a page. Similarly, `<h1>` may be followed by `<h3>` without `<h2>` between them. 
 -   Not all lists need markup. For instance, sentences that contain comma-separated lists may not need list markup ([H48: Using ol, ul and dl for lists or groups of links](http://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H48)).
+-   While an unordered list structure is often used to create navigation menus, menus can be created using other techniques such as `<nav>`. The test for Visually Apparent Lists does not include navigation menus.
```
Comment (cliffbert) [inline _baselines/13Structure.md:24]: are both lists of links (e.g. top nav bar) and dropdown menus (not comboboxes) both understood to be "navigation menus?" if so, looks ok

[the remark this answers is not in the document]
Comment (kengdoj) [reply _baselines/13Structure.md:24]: Also add under Advisory: There is not a test that programmatic lists are visually apparent lists.

**Comment by kengdoj** [comment]
Updates made. Please take another look.

**Review by cliffbert — APPROVED** [review]

**Review by drewnielson — APPROVED** [review]
Looks great!

**Review by bruce-usab — APPROVED** [review]
Very nice!

File: _baselines/ChangeLog3.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ Note: Minor punctuation, formatting and spelling changes not included.
 | 10.1 Form Names | Instruction 2: separated to 2 test instructions, added SC 1.1.1. New instruction 2 maps to SC 1.1.1 and new instruction 3 maps to SC 4.1.2 |
 | 10.1, 10.2, 10.5 | Identify Content: Added "that do not have <code>visibility:hidden</code> or <code>display:none</code>" after "Find all form components". | 
 | 10.5 Form has a Label | Test Procedure name changed to "Form has a Visible Label" and instruction 1 edited to include "has visible label(s) or instructions while the form component has focus." |
+| 13. Content Structure | Limitations, Assumptions, Exceptions: Added "While an unordered list structure is often used to create navigation menus, menus can be created using other techniques such as `<nav>`. The test for Visually Apparent Lists does not include navigation menus." |
```
Comment (kengdoj) [inline _baselines/ChangeLog3.md:21]:
```suggestion
| 13. Content Structure | Limitations, Assumptions, Exceptions: Added "A test for Visually Apparent Lists should not include navigation menus. While programmatic lists are often used to create navigation menus, menus may also be created using other techniques." |
```

**Pull request merged**